code/stat159lambda/simulations/test_correlation_simulation.py

```diff
@@ -0,0 +1,24 @@
+from __future__ import absolute_import
+from stat159lambda.simulations import correlation_simulation
+from stat159lambda.config import REPO_HOME_PATH
+
+import matplotlib
+from numpy.testing import assert_array_equal
+import unittest
+import imp
+
+try:
+    from mock import patch
+except:
+    from unittest.mock import patch
+
+
+class SimulateInterRunCorrelationTestCases(unittest.TestCase):
+    @patch.object(matplotlib.pyplot, 'savefig')
+    def test_simulate_inter_run_correlation(self, mock_sim):
```

Comment:
mock_sim should be mock_savefig since we are spying on the savefig function within sim